src/WPML_Attachment.php

```diff
@@ -62,7 +62,8 @@ public function getUrl() {
      */
     public function getIconClass() {
         if ($this->iconClass === null) {
-            $this->iconClass = $this->determine_mime_icon_class($this->getPath());
+            $mimeIconType = $this->determine_mime_icon_type($this->getPath());
```

Comment:
My intention was to keep the icon class generic (not fa-specific) in the Attachment class and only use the fa information in the rendering process (https://github.com/No3x/wp-mail-logging/blob/4bffb8b/src/WPML_Utils.php#L58). I would like to see this distinction in the WPML_Utils class.

Reply:
fair enough & agreed - like I said, feel free to diver